cmd/frontend/graphqlbackend/search.go

```diff
@@ -680,7 +680,7 @@ func resolveRepositories(ctx context.Context, op resolveRepoOp) (repoRevisions,
 			for _, vcRepoRef := range versionContext.Revisions {
 				if vcRepoRef.Repo == string(repo.Name) {
 					repoRev.Repo = repo
-					revs = append(revs, search.RevisionSpecifier{RevSpec: vcRepoRef.Ref})
+					revs = append(revs, search.RevisionSpecifier{RevSpec: vcRepoRef.Rev})
```

Comment:
Should the variable also be renamed?
```suggestion
					revs = append(revs, search.RevisionSpecifier{RevSpec: vcRepoRev.Rev})
```

Reply:
Good catch 👍